### ISSUE TITLE
Badly formatted Python enum names

### ISSUE THREAD

**Issue by lblackstone** [pulumi/pulumi-azure-native#543]
## Expected behavior
I would expect these enums to be formatted like this:
```
    AZFW_VNET = "AZFW_VNet"
    AZFW_HUB = "AZFW_Hub"
```

## Current behavior
https://github.com/pulumi/pulumi-azure-native/blob/b11bb5f60ab45161e47439cb9f8a5b5aec3ebba0/sdk/python/pulumi_azure_native/network/_enums.py#L381-L386

**Comment by mikhailshilkov** [comment]
@lblackstone Do you know if this has to be fixed in pulumi/pulumi?

**Comment by lblackstone** [comment]
I'm not sure off the top of my head. @komalali do you know?

**Comment by komalali** [comment]
Yes, it would have to be fixed in pulumi/pulumi. And it's not entirely straightforward due to how the snake casing algorithm works. Changing this can also potentially break all other python enum names, so I would tread carefully.
